lib/assist/assist_test.go

```diff
@@ -114,6 +114,51 @@ func TestChatComplete(t *testing.T) {
 	})
 }
 
+func TestClassifyMessage(t *testing.T) {
+	// Given an OpenAI server that returns a response for a chat completion request.
+	responses := []string{
+		"troubleshooting",
+		"Troubleshooting",
+		"Troubleshooting.",
+		"non-existent",
```

Comment:
Have you ever seen something like "As a Large Language Model, I think that the input can be classified as troubleshooting", or something similar? :sweat_smile:

Reply:
From the few dozen tests I ran, I never observed such a response. gpt-3.5 tend to add uppercase and dots, gpt-4 never made mistakes so far